src/Features/Core/Portable/RemoveUnnecessaryParentheses/AbstractRemoveUnnecessaryParenthesesDiagnosticAnalyzer.cs

```diff
@@ -116,9 +123,31 @@ private void AnalyzeSyntax(SyntaxNodeAnalysisContext context)
                 additionalLocations,
                 properties: null));
 
+            context.ReportDiagnostic(DiagnosticHelper.Create(
+                _unnecessaryWithoutSuggestionWithoutFadeDescriptor,
+                GetDiagnosticSquiggleLocation(parenthesizedExpression),
+                severity,
+                additionalLocations,
+                properties: null));
+
             context.ReportDiagnostic(Diagnostic.Create(
```

Comment:
each diagnostic should get a comment explaining what it is for and what the distinction is here.